install_ant-media-server.sh

```diff
@@ -245,8 +245,10 @@ if [ "$ID" == "ubuntu" ]; then
   VERSION=$(unzip -p "$ANT_MEDIA_SERVER_ZIP_FILE" ant-media-server/ant-media-server.jar  | busybox unzip -p - | grep -a "Implementation-Version"|cut -d' ' -f2 | tr -d '\r')
   if [ "$(printf '%s\n' "$REQUIRED_VERSION" "$VERSION" | sort -V | head -n1)" != "$REQUIRED_VERSION" ]; then
       check_version
-      $SUDO apt-get install libcrystalhd-dev -y
-      check
+      if [ "x86_64" == $ARCH ]; then
+        $SUDO apt-get install libcrystalhd-dev -y
```

Comment:
Please add some comments why we're making this change

Reply:
It 's my bad sorry. It is important to make this a habit and show more care in incorporating it consistently. :)